script/script.js

```diff
@@ -0,0 +1,76 @@
+const booksList = [
+  {
+    title: 'Kidagaa kimemwozea',
+    author: 'Ken Walibora',
+    id: 0,
+  },
+  {
+    title: 'Tumbo Lisiloshiba',
+    author: 'Abdimulhin Adan',
+    id: 1,
+  },
+  {
+    title: 'The River And The Source',
+    author: 'Margaret A. Ogola',
+    id: 2,
+  },
+];
```

Comment:
- [Optional] I think it would better if you put this array of objects empty instead of hard coded. in order to let the user add the books manually. right now every thing is working just fine but when we refresh the page we get those hardcoded books.